src/lib/mailgun.ts

```
@@ -0,0 +1,46 @@
+import formData from 'form-data';
+import Mailgun, { type MessagesSendResult } from 'mailgun.js';
+
+const mailgunClientSingleton = () => {
+  const mailgun = new Mailgun(formData);
+  return mailgun.client({
+    username: 'api',
+    key: process.env.MAILGUN_API_KEY as string,
+  });
+};
+
+declare const globalThis: {
```

Biome (1.9.4)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)